packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/RectangularNodeStyle.java

```diff
@@ -34,6 +34,8 @@ public final class RectangularNodeStyle implements INodeStyle {
 
     private LineStyle borderStyle;
 
+    private boolean withHeader;
```

Comment:
Should this new field be included in the `toString` result?

Reply:
It could